app/models/webhooks/slack/new_vote.rb

```diff
@@ -0,0 +1,49 @@
+class Webhooks::Slack::NewVote < Webhooks::Slack::Base
+
+  def text
+    I18n.t :"webhooks.slack.new_vote", author: author.name, position: vote_position, name: discussion_link, proposal: proposal_link(eventable)
+  end
+
+  def attachment_fallback
+    "*#{eventable.position}*\n#{eventable.statement}\n"
+  end
+
+  def attachment_title
+  end
+
+  def attachment_text
+    "#{eventable.statement}\n"
+  end
+
+  def attachment_fields
+  end
+
+  def attachment_color
+    case eventable.position
+    when "yes"
```

Comment:
We'll want to separate these out into variables somehow, because when the time comes to change a color (or allow for color customization), we don't want values like this lying around.

Looks like there's an unused `CHART_COLOURS` (hee, britishy u, go NZ!) constant in motion.rb, but I think what I'd ideally like is

```
#app/models/site_settings.rb
class SiteSettings
  def self.colors
      {
        primary: "...",
        agree: "#94D587",
        disagree: "#D1908F",
        # etc
    }.with_indifferent_access
  end
end
```

Then
```
SiteSettings.colors[:agree]
```

Later we could allow environment variables or site settings to affect those values.